learning/tour-of-beam/frontend/lib/pages/tour/screen.dart

```diff
@@ -19,33 +19,34 @@
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/svg.dart';
-import 'package:playground_components/constants/colors.dart';
 import 'package:playground_components/constants/sizes.dart';
+import 'package:playground_components/enums.dart';
 import 'package:playground_components/theme/color_provider.dart';
-import 'package:playground_components/widgets/split_view.dart';
+import 'package:playground_components/widgets/complexity.dart';
+import 'package:playground_components/widgets/split_view/controller.dart';
+import 'package:playground_components/widgets/split_view/widget.dart';
 
-import '../../components/borderless_expandable_wrapper.dart';
-import '../../components/complexity.dart';
+import '../../components/expansion_tile_wrapper.dart';
 import '../../components/page_container.dart';
 import '../../constants/sizes.dart';
 import '../../generated/assets.gen.dart';
 
-class MainScreen extends StatelessWidget {
-  const MainScreen();
+class TourScreen extends StatelessWidget {
+  const TourScreen();
 
   @override
   Widget build(BuildContext context) {
     return PageContainer(
       content: Row(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          const _Summary(),
+          const _ContentTree(),
           Flexible(
```

Comment:
`Expanded` is preferred here.

> unlike [Expanded](https://api.flutter.dev/flutter/widgets/Expanded-class.html), [Flexible](https://api.flutter.dev/flutter/widgets/Flexible-class.html) does not require the child to fill the available space.

https://api.flutter.dev/flutter/widgets/Flexible-class.html

Reply:
Removed.